setup.py

```diff
@@ -85,10 +85,12 @@ def generate_cuda() -> None:
     )
 
     if enable_sm90:
+        SM90_ALLOWED_HEAD_DIMS = {64, 128, 256}
```

Comment:
QQ When will we use 64?

Reply:
No idea. See: https://github.com/flashinfer-ai/flashinfer/blob/6e6f38d3534994c34b2c6b09b5b45c8a7b92ffd2/include/flashinfer/attention/hopper/prefill_sm90.cuh#L431